src/windows/SliderWindows/SliderView.cpp

```diff
@@ -0,0 +1,237 @@
+            else if (propertyName == "inverted") {
+                if (propertyValue.IsNull()) {
+                    this->ClearValue(xaml::Controls::Slider::IsDirectionReversedProperty());
+                }
+                else {
+                    this->IsDirectionReversed(propertyValue.AsBoolean());
+                }
+            }
+            else if (propertyName == "disabled") {
+                if (propertyValue.IsNull()) {
+                    this->ClearValue(xaml::Controls::Control::IsEnabledProperty());
+                }
+                else {
+                    this->IsEnabled(!propertyValue.AsBoolean());
+                }
+            }
+            else if (propertyName == "vertical") {
+                if (propertyValue.IsNull()) {
+                    this->ClearValue(xaml::Controls::Slider::OrientationProperty());
+                }
+                else {
+                    this->Orientation(propertyValue.AsBoolean() ?
+                        xaml::Controls::Orientation::Vertical :
+                        xaml::Controls::Orientation::Horizontal);
+                }
+            }
+            else if (propertyName == "minimumTrackTintColor") {
+                if (!propertyValue.IsNull()) {
+                    auto resDict = this->Resources();
+
+                    resDict.Insert(winrt::box_value(L"SliderTrackValueFill"), propertyValue.To<winrt::Brush>());
+                    resDict.Insert(winrt::box_value(L"SliderTrackValueFillPointerOver"), propertyValue.To<winrt::Brush>());
+                    resDict.Insert(winrt::box_value(L"SliderTrackValueFillPressed"), propertyValue.To<winrt::Brush>());
+                }
+            }
+            else if (propertyName == "maximumTrackTintColor") {
+                if (!propertyValue.IsNull()) {
+                    auto resDict = this->Resources();
+
+                    resDict.Insert(winrt::box_value(L"SliderTrackFill"), propertyValue.To<winrt::Brush>());
+                    resDict.Insert(winrt::box_value(L"SliderTrackFillPointerOver"), propertyValue.To<winrt::Brush>());
+                    resDict.Insert(winrt::box_value(L"SliderTrackFillPressed"), propertyValue.To<winrt::Brush>());
+                }
+            }
+            else if (propertyName == "thumbTintColor") {
+                if (!propertyValue.IsNull()) {
+                    auto resDict = this->Resources();
+
+                    resDict.Insert(winrt::box_value(L"SliderThumbBackground"), propertyValue.To<winrt::Brush>());
+                    resDict.Insert(winrt::box_value(L"SliderThumbBackgroundPointerOver"), propertyValue.To<winrt::Brush>());
+                    resDict.Insert(winrt::box_value(L"SliderThumbBackgroundPressed"), propertyValue.To<winrt::Brush>());
+                }
+            }
+            else if (propertyName == "thumbImage") {
+                if (!propertyValue.IsNull()) {
+                    auto imgUriString = propertyValue.AsObject()["uri"].AsString();
```

Comment:
is it guaranteed that the thumbimage is a url? i.e. it won't work with e.g. base64 encoded svg?

Reply:
I didn't see examples of base64-encoded images in the examples here, so I just made sure both local and network files render properly. I don't think it's crucial that the thumb can be an svg

Reply:
i don't think it's crucial - but maybe clarify this in the documentation that you can't use base64 SVG as an image for this

Reply:
Updated the doc to say svg won't work.